### ISSUE TITLE
Persisting a Recorded Session

### ISSUE THREAD

**Issue by pshaddel** [AndrewWalsh/openapi-devtools#22]
This tool you build saves us a ton of time! 💯 
Is it possible to persist the recorded API's somehow? I want to have the API's next time I opened the website.

**Comment by AndrewWalsh** [comment]
That's great to hear, thanks @pshaddel . You can persist the session by exporting it if that works?

**Comment by pshaddel** [comment]
@AndrewWalsh Yes, but u cannot resume what u were doing. What I am aiming for requires persisting it in somewhere like local storage. In case I close the inspection tab, or close the tab, I would like to have the recorded tabs still there.

**Comment by AndrewWalsh** [comment]
I hear you. There are multiple processes of sorts in a browser extension, each with isolated environments and specific tasks. In this case the developer tool script that runs the extension would need to communicate with a background script, which would then store this information in localstorage. I'd also need to update the UI to enable this feature, so it's a fair bit of effort. Are you ok with just saving the exported text somewhere? Even temporarily, like a VSCode tab?

**Comment by pshaddel** [comment]
Yeah, that is fine. Also I can work on it when I find time.

**Comment by AndrewWalsh** [comment]
@pshaddel I have a new version of this tool, so will close this for now https://github.com/AndrewWalsh/demystify
